### ISSUE TITLE
Plans for next release?

### ISSUE THREAD

**Issue by erip** [pytorch/fairseq#1948]
#### What is your question?

What are the plans for fairseq 0.10.0? I looked [here](https://github.com/pytorch/fairseq/milestones) but found nothing.

#### Code

N/A

#### What have you tried?

N/A

#### What's your environment?

N/A

**Comment by erip** [comment]
With the new release of PyTorch 1.5.0 and all other major PyTorch ecosystem libraries and frameworks, I wanted to circle back here. Is there a plan for a new release?

**Comment by erip** [comment]
@myleott Is there a plan here? If there's no release engineer for fairseq, maybe someone from the open-source community could help out. Since there have been so many big, breaking changes it would be nice to have discrete versions that we can point to for some of our processes.

**Comment by myleott** [comment]
Yes, this is a good point. The next release will be 1.0.0 and will hopefully be very soon. We had originally intended to do it earlier, but for a variety of reasons it kept getting delayed :/

We should bump to 1.0.0-alpha in the meantime.

Re: breaking changes, we have a more detailed changelog for 1.0.0 that we're putting together, but ultimately since 0.9.0 was so long ago, I expect most people have moved to master.
